src/stamps.ts

```diff
@@ -1,28 +1,235 @@
-export interface PostageStamps {
-  POSTAGE_STAMP?: string
+import { BeeDebug, DebugPostageBatch, BatchId } from '@ethersphere/bee-js'
+import { sleep } from './utils'
 
-  // Not implemented yet
-  POSTAGE_DEPTH?: string
-  POSTAGE_AMOUNT?: string
-  POSTAGE_USAGE?: number
+const logger = console // eslint-disable-line no-console
+
+export interface Args {
+  // Hardcoded postage stamp
+  POSTAGE_STAMP?: string /** Hardcoded postage stamp to be used (its existence, usage nor expiration is not checked) */
+
+  // Autobuy postage stamps
+  POSTAGE_DEPTH?: string /** Depth of the postage stamps to use & buy */
+  POSTAGE_AMOUNT?: string /** Amount of the postage stamps to use & buy */
+  POSTAGE_USAGE_THRESHOLD?: string /** Threshold when new postage stamp should be bought */
+  POSTAGE_USAGE_MAX?: string /** Maximal usage of the stamp to be usable by the proxy */
+  POSTAGE_TTL_MIN?: string /** Minimal TTL of the stamp to be usable by the proxy */
+  POSTAGE_REFRESH_PERIOD?: string /** How often should info about stamps be refreshed and potentially new ones be bought */
```

Comment:
Honestly, number values should be already parsed here. So for example, `depth` should be `number?` etc. This parsing should happen in `index.ts` already when reading the values from env. variables.